app/controllers/dialogs_view_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# A controller to show registered dialogs.
+class DialogsViewController < ActionController::Base
```

Comment:
SearchはAPIと画面と2つあったのでわけました。
Dialogは一つだけなので、普通にDialogsControllerで良いです。

```suggestion
class DialogsController < ActionController::Base
```

Reply:
なるほどですね。対応しました。